.github/workflows/windows.yml

```diff
@@ -22,7 +22,12 @@ jobs:
             ${{ runner.os }}-vcpkg-download-
       - name: Install libraries with vcpkg
         run: |
-          vcpkg --triplet x64-windows install readline zlib
+          vcpkg --triplet x64-windows install readline
+      - name: zlib
+        run: |
+          $zlib = 'zlib1211.zip'
+          curl -fsSL -o $zlib --retry 10 https://zlib.net/$zlib
+          &'C:\Program Files\7-Zip\7z.exe' x -osrc/ext/zlib $zlib
```

Comment:
I didn't know that `&`.

Reply:
As you've probably deduced, needed due to the space in the path.  Above does generate a zlib.dll file, which needs to be available to the build.

The install code I use generates a manifest and the needed dll's are in bin/ruby_builtin_dlls.  The mswin CI here does not run test-all or test-spec...